src/controllers/order/index.ts

```diff
@@ -0,0 +1,54 @@
+import { Body, Tags, Security, ValidateError, SuccessResponse, Response, Controller, Post, Route } from 'tsoa'
+import { Logger } from 'pino'
+
+import { DemandRequest, DemandResponse } from '../../models'
+import { logger } from '../../lib/logger'
+import { BadRequest, NotFound } from '../../lib/error-handler'
+import Database from '../../lib/db'
+import { getMemberByAddress, getMemberBySelf } from '../../lib/services/identity'
+
+const SUBTYPE = 'order'
+const STATE = 'created'
```

Comment:
No, unless you give me the reason why. I think we have talked about it. Wanted to update so they are types rather than enums. The validation does not trip and still passes.